### DISCUSSION TITLE
Release 1.2.0 Beta 12

### DISCUSSION THREAD

**Discussion by apohl79** [apohl79/audiogridder#990] [General]
## PLUGIN
- Fixed AAX installer on Windows
- Changed the audio unit type of the FX plugin back to "effect" (This is to stay backwards compatible with v1.1 on Logic Pro.)

All downloads are available at https://audiogridder.com/download/.

<hr /><em>This discussion was created from the release <a href='https://github.com/apohl79/audiogridder/releases/tag/release_1_2_0_beta12'>Release 1.2.0 Beta 12</a>.</em>

**Comment by emack999** [comment]
Just a reminder that you need to uninstall earlier plugin versions before installing Beta 12,
